specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```diff
@@ -3586,6 +3590,105 @@
         }
       }
     },
+    "FunctionsDeployment": {
+      "description": "Configuration section for Azure Function app deployment.",
+      "type": "object",
+      "properties": {
+        "storage": {
+          "description": "Storage for deployed package used by the Azure Function app.",
+          "type": "object",
+          "properties": {
+            "type": {
+              "description": "Property to select Azure Storage Type. Available option: blobContainer.",
+              "type": "string"
```

Comment:
is the implication that this is an enum? What are all the valid values, if not?

Reply:
Currently, we only plan to support `blobContainer`. However, we might support other types in the future. Based on this, should this still be an enum?

Reply:
Updated deployment storage type to be an enum.